### PR TITLE
Feat note listing

### LINKED ISSUE #12: note-listing

### PR DESCRIPTION
**Pull request by coldpigli**
What's new:
Displaying all notes on the notes page

### CODE REVIEW

**Review by nakulsharma15 — APPROVED** [review]
Everything Looks good, No changes suggested. Nice work

File: src/reducers/noteReducer.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const noteReducer = (state, action) => {
           ...state.noteData,
           title: "",
           content: "",
-          color: "",
+          color: "#FFED7D",
```
Comment (vaibhav-mahalle) [inline src/reducers/noteReducer.js:12]: you can use the color variables instead of hardcoded values.

File: src/components/NoteCard/NoteCard.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import styles from "./NoteCard.module.css";
+import DOMPurify from "dompurify";
```
Comment (vaibhav-mahalle) [inline src/components/NoteCard/NoteCard.jsx:2]: learnt something new about DOMPurify.

**Review by vaibhav-mahalle — APPROVED** [review]
learnt some new concepts while reviewing the pr. suggested a minor change.

**Pull request merged**